### DISCUSSION TITLE
New project

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#133] [Q&A] [answered]
How do I start a new Sonic Worlds Next project in Godot 4.2?

**Comment by r6915ee** [comment, marked as answer]
This is a bit of a regular Godot question, but if you want to have multiple projects using Next then I'd probably just make a copy of the unmodified version of Next. Or just make a regular new Godot project and dump all of the stuff that Next uses into that project's folder.
